index.html

```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="assets/bootstrap/bootstrap.min.css">
+  <link rel="stylesheet" href="style.css">
+  <title>Crypto Learning Course</title>
+</head>
+
+<body>
+  <div class="container-bg">
+    <div class="container">
+      <nav class="navbar navbar-expand-sm">
+  
+        <ul class="navbar-nav ml-auto">
+          <li class="nav-item">
+            <a  class="nav-link" href="twitter.com"><img src="assets/img/social/twitter.svg" alt="twitter icon"> </a>
+          </li>
+          <li class="nav-item">
+            <a  class="nav-link" href="linkedin.com"><img src="assets/img/social/Linkedin.svg" alt="Linkedin icon"> </a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">
+              Logout
+            </a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">
+              Mypage
+            </a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">
+              Korean
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </div>
+  </div>
+  
+  <header class="main-header">
+    <div class="container">
+    <nav class="main-navbar navbar-expand-sm">
+      <a href="#" class="nav-branding"><img class="branding" alt="main logo" src="assets/img/world.png"></a>
+
+      <ul class="nav-menu ml-auto">
+        <li class="nav-items close-icon hide-icon">
+          <img class="close-img " src="assets/img/close.png" alt="close">
+        </li>
+        <li class="nav-items">
+          <a href="/about.html" class="nav-links">About</a>
+        </li>
+        <li class="nav-items">
+          <a href="#teachers" class="nav-links">Teachers</a>
+        </li>
+        <li class="nav-items">
+          <a href="#partners" class="nav-links">Partners</a>
+        </li>
+        <li class="nav-items">
+          <a href="#contact" class="nav-links">Contact</a>
+        </li>
+        <li class="nav-items">
+          <a href="#getclass" class="nav-links get">Get a Class</a>
+        </li>
+      </ul>
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+    </nav>
+  </div>
+  </header>
+  <section class="head ">
+    <div class="container">
+    <div class="jumbotron">
+      <p class="hello">"Hello Crypto Lovers!"</p>
+      <h1>First Crypto Learning <br>
+        Complete Course</h1>
+      <p class="note">
```

Comment:
in your mobile version please reduce the font of the text to make sure is displays completely on the home page 
![image](https://user-images.githubusercontent.com/60612329/170841627-ed56b1c0-c4d0-4b5c-833a-8c08a9a3fd5e.png)